src/main/java/seedu/address/model/person/Person.java

```diff
@@ -57,6 +59,10 @@ public User getGitHubUser() {
         return gitHubUser;
     }
 
+    public Role getRole() {
+        return role;
+    }
```

Comment:
If we need optional type, we can implement it here, `public Optional<Role> getRole()`